test/runtests.jl

```diff
@@ -19,30 +19,59 @@ img_eps = isinteractive() ? 1e-2 : 10e-2
 #     image_comparison_facts(:gadfly, skip=[4,6,23,24,27], eps=img_eps)
 # end
 
+facts("GR") do
+    @fact gr() --> Plots.GRBackend()
+    @fact backend() --> Plots.GRBackend()
+
+    image_comparison_facts(:gr, eps=img_eps)
+end
+
 facts("PyPlot") do
     @fact pyplot() --> Plots.PyPlotBackend()
     @fact backend() --> Plots.PyPlotBackend()
 
-    image_comparison_facts(:pyplot, skip=[6,25,30,31], eps=img_eps)
+    image_comparison_facts(:pyplot, skip=[2,31], eps=img_eps)
 end
 
-facts("GR") do
-    @fact gr() --> Plots.GRBackend()
-    @fact backend() --> Plots.GRBackend()
-
-    if is_linux() && isinteractive()
-        image_comparison_facts(:gr, skip=[2,25,30,31], eps=img_eps)
+facts("PlotlyJS") do
+    @fact plotlyjs() --> Plots.PlotlyJSBackend()
+    @fact backend() --> Plots.PlotlyJSBackend()
+
+    if is_linux()
+        image_comparison_facts(:plotlyjs,
+            skip=[
+                2,  # animation
+                10, # match_dimensions not defined
```

Comment:
Not sure why `match_dimensions` isn't a keyword though. I think it just an omission from the argument lists for gr and plotlyjs. Here, it tests for it in this line https://github.com/JuliaPlots/Plots.jl/blob/master/src/backends/plotly.jl#L409 (inside the `transpose_z` function). And it obeys the keyword (it transposes a matrix in heatmap, because PyPlot (without Plots) does it differently from GR and PlotlyJS, but all Plots backends do it the same way)